tests/components/shared/form/formElements.test.js

```diff
@@ -0,0 +1,37 @@
+import { shallow } from 'enzyme';
+
+import {
+	LabeledCheckbox,
+} from 'components/shared/form';
+
+describe('Exported form elements', () => {
+	describe('LabeledCheckbox', () => {
```

Comment:
Missing tests for checking/unchecking it and checking if the event is triggered

Reply:
tests added